From 2d3903d3e151c379553db206a957776ae1f07730 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 2 Apr 2025 12:38:46 +0100
Subject: [PATCH] dtoverlays: Make piscreen consistent on swapxy

The overlay had the TI proprietary "ti,swap-xy" defined by default,
but the override for swapxy then used the generic "touchscreen-swapped-x-y"

Swap the default to use "touchscreen-swapped-x-y" to make the configuration
consistent.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/piscreen-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/piscreen-overlay.dts
+++ b/arch/arm/boot/dts/overlays/piscreen-overlay.dts
@@ -88,7 +88,7 @@
 				interrupts = <17 2>; /* high-to-low edge triggered */
 				interrupt-parent = <&gpio>;
 				pendown-gpio = <&gpio 17 GPIO_ACTIVE_LOW>;
-				ti,swap-xy;
+				touchscreen-swapped-x-y;
 				ti,x-plate-ohms = /bits/ 16 <100>;
 				ti,pressure-max = /bits/ 16 <255>;
 			};
